src/vs/workbench/electron-sandbox/window.ts

```diff
@@ -719,6 +719,43 @@ export class NativeWindow extends Disposable {
 			}
 		}
 
+		// MacOS 10.11 and 10.12 warning
+		if (isMacintosh) {
+			const majorVersion = this.environmentService.os.release.split('.')[0];
+			const eolReleases = new Map<string, string>([
+				['15', 'OS X El Capitan'],
+				['16', 'macOS Sierra'],
+			]);
+			// Refs https://en.wikipedia.org/wiki/Darwin_%28operating_system%29#Release_history
+			if (eolReleases.has(majorVersion)) {
+				const message = localize('macoseolmessage', "{0} on {1} will soon stop receiving updates. Consider upgrading your macOS version.", this.productService.nameLong, eolReleases.get(majorVersion));
+				const actions = [{
+					label: localize('macoseolBannerLearnMore', "Learn More"),
+					href: 'https://aka.ms/vscode-faq-win7'
+				}];
+
+				this.bannerService.show({
+					id: 'macoseol.banner',
+					message,
+					ariaLabel: localize('macoseolarialabel', "{0}. Use navigation keys to access banner actions.", message),
+					actions,
+					icon: Codicon.warning
+				});
+
+				this.notificationService.prompt(
+					Severity.Warning,
+					message,
+					[{
+						label: localize('learnMore', "Learn More"),
+						run: () => this.openerService.open(URI.parse('https://aka.ms/vscode-faq-win7'))
```

Comment:
Do we need a new link?

Reply:
Yup it was pending faq updates in the website.